header.php

```diff
@@ -46,7 +46,24 @@
         </ul>
       </nav>
     </div>
+   </div>
 
 <!--  <img src="http://Setupshop/wp-content/uploads/2016/09/SA.png" alt="sa-flags">  -->
 
-<?php dynamic_sidebar( 'Filters' ); ?>
\ No newline at end of file
+<!--<?php dynamic_sidebar( 'Filters' ); ?>-->
+
+<div class="container">
```

Comment:
Do you need this extra div or can you add your container class to the section and lose the wrapping div.